### PR TITLE
Feat/239/update db from extern

### PR DESCRIPTION
**Pull request by tevpg**
- #239
also:
- moves valet_hours() from tt_util to tt_conf.py (and tt_base_conf.py) because we seem to be constantly updating it and seriously, it's a config item not a core item!
  - **IMPLEMENTATION NOTE:** when pull this version at city hall, then switch to new tt_conf.py in config folder -- new tt_conf.py is called tt_conf.new.py
- lots of cgi reporting tweaks
- fullness_alert.py to pipe to mailer for alerts in case valet gets close to full
-

### CODE REVIEW

File: dbupdate.py [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+import sys
+import sqlite3
+
+import tt_dbutil as db
+import tt_util as ut
+from tt_time import VTime
+
+
+class NewVals:
+    """This tiny class is just for olding data from new source.
```
Comment (ironwoodcall) [inline dbupdate.py:25]: Does this mean to say "**h**olding"?

File: tt_dbutil.py [hunk 2]
```diff
@@ -202,4 +197,22 @@ def create_connection(db_file) -> sqlite3.Connection:
...
+def db_update(db:sqlite3.Connection,update_sql:str,commit:bool=True) -> bool:
+    """Execute a SQL UPDATE statement.  T/F indicates success.
+
+    (This could be any SQL statement, but the error checking and
+    return is based on assumption it's an UPDATE)
+    """
+    try:
+        db.cursor().execute(update_sql)
+        if commit:
+            db.commit()
```
Comment (ironwoodcall) [inline tt_dbutil.py:213]: Why not use your `db_commit()` here rather than `db.commit()`?

Comment (tevpg) [reply tt_dbutil.py:213]: db_commit was an initial idea - moved it to the dbutil module for completeness - for one thing it removes a dependency in cgi-reports to sqlite3.  I will remove db_commit in next rev?

File: dbupdate.py (hunk 1 above)
Comment (tevpg) [reply dbupdate.py:25]: Ha ha.  Yes, that would be it.  I find I make a lot of tiny data classes like this.  I don't know if that's a good thing or a bad thing.

**Pull request merged**